flocker/node/agents/blockdevice.py

```diff
@@ -143,10 +143,11 @@ class UnattachedVolume(VolumeException):
     u"The size of the underlying block device."
 )
 
-BLOCK_DEVICE_HOST = Field(
-    u"block_device_host",
+BLOCK_DEVICE_STORAGE_BACKEND_ID = Field(
```

Comment:
What myechuri said; I suggest doing it here, it'll be quick search/replace and ensure consistency across EBS and OpenStack branches.